services/app/apps/codebattle/assets/js/widgets/slices/game.js

```diff
@@ -33,6 +33,12 @@ const game = createSlice({
     setGameTask: (state, { payload: { task } }) => {
       state.task = task;
     },
+    deleteAlert: (state, { payload }) => {
+      delete state.alerts[payload];
```

Comment:
Не, не! Давай лучше через [Lodash\omit](https://lodash.com/docs/#omit) удалим значение по ключу (импортируем конкретную функцию `import omit from 'lodash/omit'`)
```suggestion
      state.alerts = omit(state.alerts, [payload]);
```